docs/conf/portland/2018/news/announcing-speakers.rst

```diff
@@ -0,0 +1,60 @@
+Happy Friday everybody! We're writing today with an update on the fast-approaching Portland conference! 
+
+First off, we're delighted to announce that we've finalized our speakers for the event! Every year talk selection gets more and more difficult as we get a larger, stronger batch of submissions. Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you! 
```

Comment:
* we've > we (ooooooh what a nit pick)
* 1 sentence per line plz <3

Reply:
fixed – and re: line breaks, I'll try to get into the habit, but I'm making no promises.